blob/fileblob/fileblob.go

```diff
@@ -16,14 +16,21 @@
 // filesystem. This should not be used for production: it is intended for local
 // development.
 //
-// Blob names are escaped using url.PathEscape before writing them
-// to disk, and unescaped using url.PathUnescape during List.
-// Exception: "/" is not escaped, so that it can be used as the real file
-// separator on the filesystem.
-// Filenames on disk that return an error for url.PathUnescape are not visible
-// using fileblob.
+// Blob keys are escaped before being used as filenames, and filenames are
+// unescaped when they are passed back as blob keys during List. The escape
+// algorithm is:
+// -- Alphanumeric characters (A-Z a-z 0-9) are not escaped.
+// -- Space (' '), dash ('-'), underscore ('_'), and period ('.') are not escaped.
+// -- The OS-specific path separator character (os.PathSeparator) is not escaped.
```

Comment:
Not quite. Slash should always map to the OS-specific path separator character.

Reply:
Done.